.gitignore

```diff
@@ -9,3 +9,4 @@ build/
 *.zip
 
 repos/
+reposense.log.0
```

Comment:
We allow 2 log files right? This line seems to ignore just one file.
May be create all log files in one `/logs` folder and ignore the entire folder?